src/sub-record/sub-record.component.html

```diff
@@ -2,8 +2,8 @@
   <table id="editor" class="table">
     <div class="field-wrapper">
       <tr *ngIf="keysByType.toggles" class="align-right">
-        <td *ngFor="let key of keysByType.toggles | addAlwaysShowToggles:schema| filterAndSortBySchema:schema; trackBy:trackByElement">
-          <slide-toggle [value]="value.get(key) | selfOrEmpty:schema.properties[key]" (valueChange)="onToggleValueChange(key, $event)"
+        <td class="value-container" *ngFor="let key of keysByType.toggles | addAlwaysShowToggles:schema| filterAndSortBySchema:schema; trackBy:trackByElement">
+          <slide-toggle [id]="'/' + key" [value]="value.get(key) | selfOrEmpty:schema.properties[key]" (valueChange)="onToggleValueChange(key, $event)"
```

Comment:
I am not sure about concating  on the template, but seems simple enough to put here to me.